docs/usermanual.md

```diff
@@ -544,3 +545,95 @@ To ensure consistent Markdown formatting run the following:
 # Use git-bash on Windows
 npm run prettier:text
 ```
+
+### Swagger
+
+In order to run Swagger, you need to execute the
+[IoT Agent](https://github.com/telefonicaid/iotagent-ul/blob/master/docs/installationguide.md#usage) and then you can
```

Comment:
```suggestion
IOT Agent (as explained [here](installationguide.md#usage) and then you can
```

Reply:
fixed in 2808c0b11be011d26b2a3c9e0077e1c4a22c056c